neclib/devices/ad_converter/cpz3177.py

```diff
@@ -7,6 +7,57 @@
 
 
 class CPZ3177(ADConverter):
+
+    """a/d converter, which can convert by 32 or 64 channels.
+
+    Notes
+    -----
+    Configuration items for this device:
+
+    rsw_id : {0, 1, ..., 16} or {"0", "1", ..., "9", "A", ..., "F"}
+        Board identifier. This should be set to the same value as the rotary switch
+        "RSW1" mounted on the side of the board. The board is shipped with default RSW1
+        setting of 0. This ID would be non-zero, when multiple PCI board of same model
+        are mounted on a single FA (Factory Automation) controller.
+
+    ave_num : int
+        Sampling data number to calculate average of data. This source code not only
+        measure the actual voltage, but also calculate the average of data
+        in order to measure a faint voltage. It should not be define too large number
+        against sampling frequency.
```

Comment:
```suggestion
        Number of sampled data to be averaged over. This script not only measures the
        actual voltage, but also performs fluctuation reduction in order to measure a
        faint voltage. The value should not be too large compared to sampling frequency
        ``smpl_freq``, otherwise the output won't follow the change of voltage.
```

値が大きすぎるとよくない理由はこれで良かったでしょうか？